setup.py

```diff
@@ -7,7 +7,9 @@
 
 dependencies = [
     "clvm>=0.9.2",
-    "clvm_tools_rs>=0.1.37"
+    "clvm_tools_rs>=0.1.37",
+    "importlib_metadata~=6.11.0",
```

Comment:
You want to block `6.12`?  `~` only let's the last specified segment increase and locks all others.

Also, can't we just use `importlib.metadata` as it was added in 3.8 which is our minimum?  https://docs.python.org/3.12/library/importlib.metadata.html#module-importlib.metadata  Though there are arguments for just always using the backports libraries as well rather than thrashing depending on needing a new feature from them or not.

Reply:
Changed to use `importlib.metadata`